src/testdir/test_terminal.vim

```diff
@@ -2370,3 +2370,18 @@ func Test_terminal_in_popup()
   call StopVimInTerminal(buf)
   call delete('XtermPopup')
 endfunc
+
+func Test_issue_5607()
+  if !has('win32')
```

Comment:
https://github.com/vim/vim/issues/5607#issuecomment-583945615
It seems that this is not Windows-specific.

Reply:
Fixed.